UIKitHw/Models/UserSettings.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  UserSettings.swift
+//  UIKitHw
+//
+//  Created by Aleksandr Dorofeev on 04.10.2022.
+//
+
+import Foundation
+
+/// Additional class with users collection.
+final class UserSettings {
+
+  // MARK: - Public properties.
+  // Singletone.
+  static var shared = UserSettings()
+  public lazy var defaults = UserDefaults.standard
+  public var users: [User] = []
+  
+  // MARK: - Public methods.
+  public func saveMail(mail: String, forKey: String) {
```

Comment:
```suggestion
  func saveMail(mail: String, forKey: String) {
```
public  не используем